### PR TITLE
Support setting timeouts on client

### PR DESCRIPTION
**Pull request by hmac**
That

### CODE REVIEW

File: lib/companies_house/client.rb [hunk 1]
```diff
@@ -16,6 +16,8 @@ def initialize(config)
       raise ArgumentError, "Missing API key" unless config[:api_key]
       @api_key = config[:api_key]
       @endpoint = URI(config[:endpoint] || ENDPOINT)
+      @open_timeout = config[:open_timeout] || 60
+      @read_timeout = config[:read_timeout] || 60
```
Comment (hmac) [inline lib/companies_house/client.rb]: These are the default values for `Net::HTTP` - thought it better to be explicit.

**Comment by hmac** [comment]
@acraileanu I've added a new error type for timeouts - could you take another look?

File: lib/companies_house/timeout_error.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module CompaniesHouse
+  # Specific error class for when an invalid API key is used to access the service
```
Comment (acraileanu) [inline lib/companies_house/timeout_error.rb]: We need to update this message.

Comment (hmac) [reply lib/companies_house/timeout_error.rb]: 🙈 good catch

**Pull request merged**